Release v0.13.1

ksedgwic opened the pull request:
Checklist:
- [x] `v0.13.1rc1`
- [x] decide on no-private
- [x] stable

ksedgwic committed:
Updated the Changelog

ksedgwic committed:
Updated configure version string.

Also changed the maintainer address at ZmnSCPxj's request.

vincenzopalazzo reviewed (approved):
ACK https://github.com/ZmnSCPxj/clboss/pull/191/commits/e4f15b6aeccc0548b8c80f7948d2265111c4e9c5

ksedgwic committed:
remove references to the "private" field of listpeers channel info

Fixes issue ([#193])

Changelog-Fixed: Removed references to deprecated "private" field of channel info returned from API.
